plat/common/x86/cpu_native.c

```diff
@@ -48,6 +48,11 @@ unsigned long read_cr2(void)
 
 void system_off(void)
 {
+#if CONFIG_KVM_VMM_FIRECRACKER
+	/* Trigger a reset via the PS/2 controller */
+	outb(0x64, 0xFE);
```

Comment:
I don't think this is Firecracker specific. I guess this is just one of the legacy ways of pulsing the CPU's reset pin thruogh the PS/2 Controller? Pretty sure this should work on QEMU as well. If anything,  I would personally just place it at the end of the function without any ifdef's, as a last resort, following the `isa-debug-exit`.

Reply:
The problem is that it will cause a reset instead of a shutdown on non-firecracker systems. While QEMU has a command-line configuration for that (`-no-reboot`), it's not the default. 
And poking random I/O registers seems like a bad idea. So maybe it makes more sense to put the QEMU ACPI/ISA stuff in an `#ifdef`, too.

Reply:
On firecracker it is a power off and not a reset trigger. So putting it in a reset function would be wrong.

Reply:
This inconsistency is unfortunate :/. Yeah I guess an ifdef should do.